openvdb_points/tools/PointAttribute.h

```diff
@@ -301,18 +303,21 @@ inline void appendAttribute(PointDataTree& tree,
 
```

Comment:
have we decided not to expose a free function for appendAttribute() to take a name and type? i.e, for a vector and name and type pairs, I should do this myself/do it on the fly per leaf?